docs/DeveloperGuide.adoc

```diff
@@ -77,6 +77,42 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Rename Tag
+
+*MSS*
+
+. User requests to rename tag
+. AddressBook returns and shows a list of unique tags
+. User requests to select a specific tag in the list
+. User requests to rename a specific tag in the list
```

Comment:
1. You can combine the above two. OR
2. You need to have a step where the `AddressBook` does something.
Remember, it was mentioned that the use case should show the interaction between the user and the system?

Reply:
Okay! What about this? 

. AddressBook returns and shows a list of unique tags
. User requests to select a specific tag in the list
. AddressBook confirms selection and prompts for an action
. User requests to rename a specific tag in the list